tests/pages/services/ServiceFormModal-cy.js

```diff
@@ -1648,5 +1648,75 @@ describe("Service Form Modal", function() {
         });
       });
     });
+
+    context("Multi-container (pod)", function() {
+      beforeEach(function() {
+        cy
+          .get(".menu-tabbed-item-label")
+          .eq(0)
+          .click()
+          .get(".menu-tabbed-view h2")
+          .contains("Service");
+      });
+
+      it("Should add new container", function() {
+        cy.get(".menu-tabbed-view .button.button-primary-link").eq(3).click();
+      });
+
+      it("Should contain two containers navigation under Services tab", function() {
+        cy.get(".menu-tabbed-view .button.button-primary-link").eq(3).click();
+
+        cy
+          .get(".menu-tabbed-item-label")
+          .eq(0)
+          .siblings()
+          .should("to.have.length", 2);
```

Comment:
If we really want to test that clicking that button adds a container we should verify the initial state first.
Like before clicking the button in should `'to.have.length', 1`

Looks like we can leverage chai's `change` assertion 
```javascript
expect(function() {
  cy.get(".menu-tabbed-view .button.button-primary-link").eq(3).click();
}).to.change(cy.get(".menu-tabbed-item-label").eq(0).siblings(), 'length')
``` 

or better `increases`

```javascript
expect(function() {
  cy.get(".menu-tabbed-view .button.button-primary-link").eq(3).click();
}).to.increase(cy.get(".menu-tabbed-item-label").eq(0).siblings(), 'length')
``` 

https://docs.cypress.io/docs/making-assertions#section-available-assertions